README.md

```diff
@@ -136,6 +136,9 @@ await ofetch("http://google.com/404", {
 });
 ```
 
+> [!NOTE]
```

Comment:
I think we can use a normal paragraph since no user action is involved...

Reply:
Good point!